specs/src/specs/block_validity_rules.md

```diff
@@ -41,7 +41,9 @@ Celestia specifc validity rules can be categorized into two groups:
 
 All `BlobTx` transactions must be valid according to the [BlobTx validity rules](../../../x/blob/README.md#validity-rules)
 
-All remaining transaction types do not have to by valid if included in a block. For a complete list of modules see [state machine modules](./state_machine_modules.md).
+All remaining transaction types do not have to be valid if included in a block. In other words: random tx data is permitted, but decodable sdk.Txs must pass all [AnteHandler](./ante_handler.md) checks in order for the block to be valid.
```

Comment:
Note that in `CheckTx`, we actually try to decode the txs as sdk txs else we error:

https://github.com/celestiaorg/celestia-app/blob/cbe5020e35d17bb26da298b9bb14c16ac4fb06ba/app/check_tx.go#L21

Reply:
Good catch! Resolved via 6e54aaa0ee5027cd04b8c845cf8d4eeacc2d8340